section4/exercises/classMethods.js

```diff
@@ -28,13 +28,35 @@ Make sure to run the file with node in your command line.
 // that the proteins have been changed.
 
 class Burrito {
-  constructor(protein, base) {
+  constructor(protein, base, toppings) {
     this.protein = protein;
     this.base = base;
-  }
+    this.toppings = toppings;
+
+  };
+  changeProtein(changeProtein) {
+    this.protein = changeProtein;
+  };
+  addTopping(addTopping) {
+    this.toppings.push(addTopping);
```

Comment:
Good job with dynamic methods, and finding a prototype method to work with an array. I think experimenting with prototype methods in a repl is a great way to practice before Turing starts.